model/kubernetes/kubernetes_nodepool.rb

```diff
@@ -0,0 +1,28 @@
+#  frozen_string_literal: true
+
+require_relative "../../model"
+
+class KubernetesNodepool < Sequel::Model
+  one_to_one :strand, key: :id
+  many_to_one :kubernetes_cluster
```

Comment:
Since both of them are in the `Kubernetes` context, `cluster` might be more readable. 
I prefer `kubernetes_nodepool.cluster` instead of `kubernetes_nodepool.kubernetes_cluster`

similar discussion: https://github.com/ubicloud/ubicloud/pull/2628#discussion_r1920043782